src/Spice86/Views/CpuStackView.axaml

```diff
@@ -0,0 +1,40 @@
+<UserControl
+    x:Class="Spice86.Views.CpuStackView"
+    xmlns="https://github.com/avaloniaui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:avaloniaHex="clr-namespace:AvaloniaHex;assembly=AvaloniaHex"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:dialogHost="clr-namespace:DialogHostAvalonia;assembly=DialogHost.Avalonia"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:rendering="clr-namespace:AvaloniaHex.Rendering;assembly=AvaloniaHex"
+    xmlns:viewModels="clr-namespace:Spice86.ViewModels"
+    xmlns:userControls="clr-namespace:Spice86.UserControls"
xmlns:controls="using:Spice86.Controls"
+    d:DesignHeight="450"
+    d:DesignWidth="800"
+    x:CompileBindings="True"
+    x:DataType="viewModels:CpuStackViewModel"
+    mc:Ignorable="d">
+	<ScrollViewer>
+		<Grid RowDefinitions="Auto,Auto,*">
+			<TextBlock Grid.Row="0" Grid.RowSpan="3" IsVisible="{Binding !IsPaused}" Text="Stack memory is displayed only when the emulator is paused" />
+			<StackPanel Grid.Row="1" Orientation="Horizontal">
+				<Label>Physical Address:</Label>
+				<TextBlock Margin="5,0,0,0" Text="{Binding PhysicalAddress}" />
+			</StackPanel>
+			<avaloniaHex:HexEditor
+				Grid.Row="2"
+				x:Name="HexViewer"
+				Document="{Binding DataMemoryDocument}"
+				IsVisible="{Binding IsPaused}">
+				<TextElement.FontFamily>
+					<OnPlatform Default="{StaticResource RobotoMonoFont}" />
+				</TextElement.FontFamily>
+				<avaloniaHex:HexEditor.Columns>
+					<rendering:OffsetColumn />
+					<rendering:HexColumn />
+					<rendering:AsciiColumn />
+				</avaloniaHex:HexEditor.Columns>
+			</avaloniaHex:HexEditor>

```

Comment:
Can you add the Copy/Show in structureview context menu here too?

Reply:
Sure. In order to not duplicate XAML code, I think this will CpuStackViewModel will have both a MemoryViewModel, and the CpuStackViewModel itself will serve this kind of view in a first TabControl:

![373889212-d0933287-d24c-4fbe-8cfc-947fd851fbcb](https://github.com/user-attachments/assets/93f37be2-1cd1-4b22-aa58-4c8f356a2516)

See the origin of the idea here: https://github.com/epasveer/seer/issues/127

Reply:
sounds good